x-pack/plugin/esql/qa/testFixtures/src/main/java/org/elasticsearch/xpack/esql/CsvTestsDataLoader.java

```diff
@@ -364,11 +395,16 @@ private static void loadCsvData(
                 }
             }
         }
-        if (builder.length() > 0) {
+        if (builder.isEmpty() == false) {
```

Comment:
This is a superficial change that ends up doing the same call as before but with a different comparison:
`(builder.length() == 0) == false`

Reply:
This came from Intellij warnings. I often do 'refactoring' to reduce warnings in Intellij when working on code that has warnings. Ideally I'd like code to have no warnings.